beacon.go

```diff
@@ -133,20 +147,26 @@ func validateServerConfig() error {
 	return nil
 }
 
-func parseInput(r *http.Request) (string, string, int64, error) {
+type queryParams struct {
+	refName string
+	allele  string
+	coord   int64
+}
+
+func parseInput(r *http.Request) (queryParams, error) {
```

Comment:
Usually return a pointer so that you can just return nil [callers shouldn't use the value when you have an error anyway, so it isn't really helping them]

Reply:
done